frontend/public/locales/en/renew-adult-child.json

```diff
@@ -282,6 +282,7 @@
         "date-of-birth-year-number": "Year must be a number, for example 1950",
         "date-of-birth-year-required": "Date of birth must include a year",
         "client-number-required": "Enter child's client number",
+        "client-number-valid": "Must be a valid Client Number. Your input must be 11 or 13 digits long.",
```

Comment:
Might be a nitpick, but shouldn't this be `client number` (instead of `Client Number`)? Everywhere else that I can see it's all lowercase.

Reply:
I copy and pasted the text from `status.json`, the error message is "Must be a valid Application Code or Client Number. Your input must be 6, 11 or 13 digits long." Where both Application Code and Client Number are upper case. 

I'll change that as well, it's also lower case in French